assets/json/assets.json

```diff
@@ -4,78 +4,78 @@
     },
     "core": {
-        "file":   "textures/coreAnimation_v2.png"
+      "file": "textures/coreAnimation_v2.png"
     },
     "innerRing": {
-        "file":   "textures/innerLayerAnimation_v2.png"
+      "file": "textures/innerLayerAnimation_v2.png"
     },
     "unlockedOuterRing": {
-        "file":   "textures/smallOuterLayerAnimation_v4.png"
+      "file": "textures/smallOuterLayerAnimation_v4.png"
     },
     "playerProgress": {
-        "file":   "textures/playerProgressAnimation_v2.png"
+      "file": "textures/playerProgressAnimation_v2.png"
     },
     "fog": {
-        "file":   "textures/fog.png"
+      "file": "textures/fog.png"
     },
     "lens_flare": {
-        "file":   "textures/flare.png"
+      "file": "textures/flare.png"
     },
     "meteor_shower_standalone": {
-        "file":   "textures/meteor_shower_standalone.png"
+      "file": "textures/meteor_shower_standalone.png"
     },
     "shooting_star_standalone": {
-        "file":   "textures/shooting_star_standalone.png"
+      "file": "textures/shooting_star_standalone.png"
     },
     "fog_standalone": {
-        "file":   "textures/fog_standalone.png"
+      "file": "textures/fog_standalone.png"
     },
     "greyscale_standalone": {
-        "file":   "textures/greyscale_standalone.png"
+      "file": "textures/greyscale_standalone.png"
     }
   },
   "sounds": {
-      "game": {
-          "type":     "sample",
-          "file":     "sounds/background_music_v1.ogg",
-          "stream":   true,
-          "volume":   0.5
-      },
-      "fog": {
-          "type":     "sample",
-          "file":     "sounds/fog_sound_v1.wav",
-          "volume":   0.5
-      },
-      "grayscale": {
-          "type":     "sample",
-          "file":     "sounds/grayscale_sound_v1.wav",
-          "volume":   0.5
-      },
-      "meteor": {
-          "type":     "sample",
-          "file":     "sounds/meteor_sound_v1.wav",
-          "volume":   0.5
-      },
-      "shootingStar": {
-          "type":     "sample",
-          "file":     "sounds/shooting_star_sound_v1.wav",
-          "volume":   0.5
-      }
+    "game": {
+      "type": "sample",
+      "file": "sounds/background_music_v1.ogg",
+      "stream": true,
+      "volume": 0.5
+    },
+    "fog": {
+      "type": "sample",
+      "file": "sounds/fog_sound_v1.wav",
+      "volume": 0.5
+    },
+    "grayscale": {
+      "type": "sample",
+      "file": "sounds/grayscale_sound_v1.wav",
+      "volume": 0.5
+    },
+    "meteor": {
+      "type": "sample",
+      "file": "sounds/meteor_sound_v1.wav",
+      "volume": 0.5
+    },
+    "shootingStar": {
+      "type": "sample",
+      "file": "sounds/shooting_star_sound_v1.wav",
+      "volume": 0.5
+    }
   },
   "widgets": {
     "menubutton": "widgets/menubutton.json",
     "clearbutton": "widgets/clearbutton.json"
   },
   "fonts": {
-    "saira32": {
-      "file": "fonts/SairaSemicondensed.ttf",
+    "gillsans32": {
```

Comment:
Is there a reason you changed the font for the game?

Reply:
I thought the Designers wanted to update the font to the new font. I can revert to the old font if needed.

Reply:
Yes, Richard is right. Please use the new gill sans font.